bsp/boards/radio.h

```diff
@@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
 void     radio_rfOn(void);
 void     radio_rfOff(void);
 // TX
-void     radio_loadPacket(uint8_t* packet, uint8_t len);
+void     radio_loadPacket(uint8_t* packet, uint16_t len);

```

Comment:
If you change the signature, you need to change the BSP of all boards. You must have received an e-mail from Travis telling you the builds failed. It's a small change, and I suggest you fix the BSPs of the different boards, until Travis passes again.